README.md

```diff
@@ -43,6 +43,9 @@ through plugins.
   Kubernetes.
 - **Dynamic Load Balancing**: Load balance traffic across multiple backend
   services.
+- **Hash-based Load Balancing**: Load balance consisten hashing/sticky sessions.
```

Comment:
(Looks like you might want to enable spell checking on your editor :)

"consisten" -> "consistent"